js/slider.js

```diff
@@ -0,0 +1,71 @@
+import { PRICES, MAX_PRICE } from './settings.js';
+
+const SLIDER_DEFAULT_SETTINGS = {
+  step: 100,
+  connect: 'lower',
+  format: {
+    to: function (value) {
```

Comment:
```suggestion
    to: (value) => {
```
у тебя везде стрелочные функции, так что будет будет так везде)